options_test.go

```diff
@@ -18,6 +18,7 @@ func TestResolveOptions(t *testing.T) {
 				separator: '\t',
 				scaleType: linear,
 				chartType: undefinedChartType,
+				colorType: defaultColor,
```

Comment:
Did you try removing these `colorType: defaultColor` lines? I think it should be equivalent. Because you've "made the zero value useful" (https://www.youtube.com/watch?v=PAAkCSZUG1c&t=6m25s)